lib/debug.dl

```diff
@@ -2,7 +2,9 @@
  * an external debugger tool.
  */
 
-extern function debug_event(operator_id: (u32, u32, u32), w: DDWeight, ts: 'T1, input1: 'A1, out: 'A2): ()
-extern function debug_event_join(operator_id: (u32, u32, u32), w: DDWeight, ts: 'T1, input1: 'A1, input2: 'A2, out: 'A3): ()
+extern function debug_event(operator_id: (u32, u32, u32), w: DDWeight, ts: 'T1,
```

Comment:
In one of future commits, can we add a typedef to `debug.dl`, e.g.,
```
typedef DDlogOpId = DDlogOpId {
  rel: u32,
  rule: u32,
  op: u32
}
```

Reply:
On a second thought, since this will be serialized into every debugger message, maybe we should keep it as a tuple for more concise output:
```
typedef DDlogOpId = (u32, u32, u32)
```